docs/ref/databases.txt

```diff
@@ -439,6 +439,24 @@ more details.
 .. _MySQL option file: https://dev.mysql.com/doc/refman/5.6/en/option-files.html
 .. _MySQLdb documentation: http://mysql-python.sourceforge.net/
 
+.. _mysql-sql-mode:
+
+Setting ``sql_mode``
+~~~~~~~~~~~~~~~~~~~~
+
+From MySQL 5.7 onwards, the default value of the ``sql_mode`` option contains
+``STRICT_TRANS_TABLES``. That option is escalating warnings into errors when
+data are truncated upon insertion, so Django highly recommends activating a
+`strict mode`_ for MySQL to prevent data losses (either ``STRICT_TRANS_TABLES``,
+``STRICT_ALL_TABLES``, or ``TRADITIONAL``).
+
+.. _strict mode: https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-strict
+
+If you are using an older version of MySQL, you can set the ``sql_mode`` like
```

Comment:
I understand the intent but I guess "If you are using an older version of MySQL" isn't really needed since the instructions work on any version (maybe you want to say something like "If you need to customize the SQL mode, you can set it like...").